cmd/reindex/main.go

```diff
@@ -485,6 +485,7 @@ func convertToSearchDataModel(searchDataImport extractorModels.SearchDataImport)
 		Published:       searchDataImport.Published,
 		Survey:          searchDataImport.Survey,
 		Language:        searchDataImport.Language,
+		CanonicalTopic:  searchDataImport.CanonicalTopic,
```

Comment:
It's worth ordering fields alphabetically when creating new variable `searchDIM`